src/components/Card.js

```diff
@@ -10,19 +10,23 @@ class Card extends Component {
         const { name, description, from } = this.props.kudos
         return (
             <StyledCard>
-                <Row>
+                <Row margin={'0 0 8px'}>
                     <h6>
                         <KudosTitle>KUDOS </KudosTitle>
                         for
-                        <KudosMan slackUser={this.isSlackUser(name)}> {name}</KudosMan>
+                        <KudosMan slackUser={this.isSlackUser(name)}> {name[0] === '@' ? name.substring(1, name.length) : name}</KudosMan>
```

Comment:
I would drop `slack` part of the user. Frontend app does not need to know the domain. Backend may use users from gmail company account, trello or anything - i.e. frontend app does not have to be restricted to slack users.

Reply:
Maybe we don't need now any division of users, because now we have only Slack users, right?

Reply:
I removed all "slack user" logic :)

Reply:
👍